### PR TITLE
Im/hugo update

### PR DESCRIPTION
**Pull request by IanMadd**
### Description

This updates the Hugo site:
- Adds changes from Tim's Workstation PR
- Adds hidden resource pages
- Other content changes from other recent PRs

### Issues Resolved

[List any existing issues this PR resolves, or any Discourse or
StackOverflow discussion that's relevant]

### Check List

- [ ] Spell Check
- [X] Local build
- [X] Examine the local build
- [ ] All tests pass

### CODE REVIEW

**Review by schisamo — APPROVED** [review]

**Comment by kagarmoe** [comment]
We don't have the github link at the top of the Resources entries. We might want to add one that directs to `issues` instead of `pulls`, since these are generated.

File: content/install_workstation.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
++++
```
Comment (kagarmoe) [inline content/install_workstation.md:1]: We will need a redirect from install_dk

Comment (IanMadd) [reply content/install_workstation.md:1]: Let me fix this in the next PR. I'll add all the redirects that are currently in Sphinx.

**Comment by kagarmoe** [comment]
Let's add some words about building the Hugo site & particularly about `make clean_all` to the Readme.md.

**Comment by kagarmoe** [comment]
I'd like to increase the space between the bottom of code-blocks and the next header. It seems too cramped:
Screen Shot 2020-02-07 at 9 41 58 AM

(from http://localhost:1313/resources/template/)

**Pull request merged**